core/src/main/scala/kafka/raft/RaftManager.scala

```diff
@@ -95,15 +95,15 @@ trait RaftManager[T] {
 }
 
 class KafkaRaftManager[T](
-  nodeId: Int,
-  baseLogDir: String,
   recordSerde: RecordSerde[T],
   topicPartition: TopicPartition,
   config: KafkaConfig,
   time: Time,
   metrics: Metrics
 ) extends RaftManager[T] with Logging {
 
+  private val nodeId = config.brokerId
+  private val baseLogDir = config.logDirs.head
```

Comment:
this isn't quite correct since we could also have a specifically configured metadata.log.dir

Reply:
Right. There are a few placeholders in here until we get all of the configs in.